src/python/zquantum/core/graph.py

```diff
@@ -73,163 +66,223 @@ def generate_graph_node_dict(graph: nx.Graph) -> dict:
     return nodes_dict
 
 
+Sampler = Iterator[float]
+
+
+def uniform_sampler(min_value=0, max_value=1) -> Sampler:
+    while True:
+        yield random.uniform(min_value, max_value)
+
+
+def constant_sampler(value) -> Sampler:
```

Comment:
```suggestion
def constant_sampler(value: float) -> Sampler:
```
Maybe we should we make sure this value is in [0,1].

Reply:
This sampler just produces `value` again and again, this doesn't have to be a probability, so I'd leave it unbounded.